routers/api/packages/nuget/nuget.go

```diff
@@ -111,13 +112,14 @@ func getSearchTerm(ctx *context.Context) string {
 
 // https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedQueryBuilder.cs
 func SearchServiceV2(ctx *context.Context) {
-	skip, take := ctx.FormInt("skip"), ctx.FormInt("take")
+	skip, take := ctx.FormInt("$skip"), ctx.FormInt("$top")
 	if skip == 0 {
-		skip = ctx.FormInt("$skip")
+		skip = ctx.FormInt("skip")
```

Comment:
```suggestion
		skip = ctx.FormInt("skip")
```
?  should be `$skip` or `skip`